app/src/main/java/org/standardnotes/notes/LoginActivity.kt

```diff
@@ -57,7 +57,7 @@ class LoginActivity : AppCompatActivity() {
                 hideProgress()
             }
 
-            override fun onFailure(call: Call<SigninResponse>, t: Throwable) {
+            override fun onFailure(call: Call<SigninResponse>?, t: Throwable?) {
```

Comment:
I see. why not just pass a nullable `onCancel` in showConfirmationAlert?

also it seems like bad architecture to keep the show error dialog stuff on crypt. but that's a discussion for a later time.